client/web/src/components/ConnectionPopover.scss

```
@@ -37,19 +25,13 @@
         &:empty {
             display: none;
         }
-        .theme-redesign & {
-            border-top: solid 1px var(--border-color-2);
-            padding-top: 0.25rem;
-            padding-bottom: 0.25rem;
-        }
+        border-top: solid 1px var(--border-color-2);
+        padding-top: 0.25rem;
+        padding-bottom: 0.25rem;
```

FYI, this is also overwritten by `FilteredConnection`, a quick fix would be this:

```suggestion
        padding-top: 0.25rem !important;
        padding-bottom: 0.25rem !important;
```

Unless there's a better approach, these classes are used in a few places so not fully confident on a reliable way to increase specificity here. Maybe add an element selector on `ul`?